src/ppu.jakt

```diff
@@ -421,7 +421,13 @@ class PPU {
         }
         for sprite in (0u8..63u8).inclusive() {
             let sprite_addr = sprite * 4u8
-            let sprite_y = .oam[sprite_addr] + 1
+            mut sprite_y = 0u8
+            //https://www.nesdev.org/wiki/PPU_OAM
+            // Byte 0 - Hide a sprite by writing any values in $EF-$FF here.
+            // If "any values" can go to $ef-$ff, we only care what's below $ee
+            if (.oam[sprite_addr] < 0xee ) {
+                sprite_y = .oam[sprite_addr] + 1
```

Comment:
I think you want:

```
if .oam[sprite_addr] > 0xee {
  continue
}
```
instead.

That will effectively make those sprites invisible, which I think is how the docs explain it.